webtau-core/src/main/java/com/twosigma/webtau/reporter/WebTauTestMetadata.java

```diff
@@ -17,27 +17,35 @@
 package com.twosigma.webtau.reporter;
 
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
-public class WebTauTestMeta {
+public class WebTauTestMetadata {
     private final Map<String, Object> meta;
 
-    public WebTauTestMeta() {
+    public WebTauTestMetadata() {
         meta = new LinkedHashMap<>();
     }
 
     /**
      * register key/value meta
-     * @param key key
-     * @param value value
-     * @return previously registered value for key
+     * @param values metadata values to add
+     * @return previously registered values
```

Comment:
remind me what we need the previous values for?

Reply:
To potentially fail hard in case of custom metadata wrappers.